.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
 - 10.16.3
-script: ng build --project ng-bootstrap-form-validation
+script: ng build --prod --project ng-bootstrap-form-validation
```

Comment:
Hrm, _should_ we update the `build` script in `package.json` instead?

Reply:
Wouldn't be a bad idea

Reply:
I misread this, I already updated it in the previous MR